frappe/integrations/doctype/google_contacts/google_contacts.js

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2019, Frappe Technologies and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Google Contacts', {
+	refresh: function(frm) {
+		frm.set_value("user", frappe.session.user);
+
+		if (frm.doc.refresh_token) {
+			frm.add_custom_button(__('Sync Contacts'), function () {
+				frappe.show_alert({
+					indicator: 'green',
+					message: __('Syncing')
+				});
+				frappe.call({
+					method: "frappe.integrations.doctype.google_contacts.google_contacts.sync",
+					args: {
+						"g_contact": frm.doc.name
+					},
+				}).then((r) => {
+					frappe.hide_progress();
+					frappe.msgprint(__("{0}", [r.message]));
```

Comment:
This doesn't translate the message. Either trigger the msgprint from the server or write the message here.